### DISCUSSION TITLE
bug report

### DISCUSSION THREAD

**Discussion by rohit-paygude** [trendmicro/cloudone-filestorage-plugins#57] [General]
Hi,

I have found a bug in cloudone-filestorage-plugins/post-scan-actions/aws-python-promote-or-quarantine/handler.py
Line number 147 and 148 is causing a following error:
[ERROR] Runtime.UserCodeSyntaxError: Syntax error in module 'handler': invalid syntax (handler.py, line 147)
Traceback (most recent call last):
  File "/var/task/handler.py" Line 147
                f'{FSS_TAG_PREFIX}scan-detail-code': str(code),

To fix it, you need to update it to following:
            f'{FSS_TAG_PREFIX}scan-detail-code={str(code)}',
            f'{FSS_TAG_PREFIX}scan-detail-message={urllib.parse.quote_plus(CODE_MESSAGES.get(code, CODE_MESSAGES[CODE_MISC]))}',

**Comment by carpusherw** [comment]
Hi @rohit-paygude ,
Thank you for reporting and fixing the issue. We appreciate your contribution.

**Reply by carpusherw** [reply]
Issue #59 created for the record.
